### PR TITLE
Payment screen showing different UI if there is a saved credit card

### PR DESCRIPTION
**Pull request by RobinCsl**

### CODE REVIEW

File: app/MMB/src/scenes/tripServices/insurance/paymentScene/CardOption.js [hunk 1]
```diff
@@ -0,0 +1,89 @@
+// @flow
+
+import * as React from 'react';
+import { View, TouchableWithoutFeedback, Image } from 'react-native';
+import { type TranslationType } from '@kiwicom/mobile-localization';
+import { TextIcon, StyleSheet, Text } from '@kiwicom/mobile-shared';
+import { defaultTokens } from '@kiwicom/mobile-orbit';
+
+type Props = {|
+  +active: boolean,
```
Comment (tbergq) [inline app/MMB/src/scenes/tripServices/insurance/paymentScene/CardOption.js]: should be isActive

File: app/MMB/src/scenes/tripServices/insurance/paymentScene/CardOption.js [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+  +onPress: () => void,
+  +content: TranslationType,
```
Comment (tbergq) [inline app/MMB/src/scenes/tripServices/insurance/paymentScene/CardOption.js]: what is this content? Maybe the name could describe more what it is?

File: app/MMB/src/scenes/tripServices/insurance/paymentScene/CardOptions.js [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+  onPressAnotherCard = () => {
+    this.props.setActiveCard('ANOTHER_CARD');
+  };
+
+  onPressSavedCard = () => {
+    this.props.setActiveCard('SAVED_CARD');
+  };
+
+  render() {
+    const savedCardActive = this.props.active === 'SAVED_CARD';
```
Comment (tbergq) [inline app/MMB/src/scenes/tripServices/insurance/paymentScene/CardOptions.js]: isSavedCardActive

File: app/MMB/src/scenes/tripServices/insurance/paymentScene/CardOptions.js [hunk 4]
```diff
@@ -0,0 +1,105 @@
...
+    const anotherCardActive = this.props.active === 'ANOTHER_CARD';
```
Comment (tbergq) [inline app/MMB/src/scenes/tripServices/insurance/paymentScene/CardOptions.js]: isAnotherCardActive

File: app/MMB/src/scenes/tripServices/insurance/paymentScene/PaymentScene.js [hunk 5]
```diff
@@ -1,58 +1,88 @@
...
+|};
+
+export default class PaymentScene extends React.Component<{||}, State> {
+  state = {
+    card: null,
+    active: 'ANOTHER_CARD',
+  };
+
+  componentDidMount() {
+    getCard().then(card => {
```
Comment (tbergq) [inline app/MMB/src/scenes/tripServices/insurance/paymentScene/PaymentScene.js]: could this be async/await please

File: app/MMB/src/scenes/tripServices/insurance/paymentScene/PaymentScene.js [hunk 6]
```diff
@@ -1,58 +1,88 @@
...
+  setActiveCard = (pressed: 'SAVED_CARD' | 'ANOTHER_CARD') => {
+    if (this.state.active !== pressed) {
+      this.setState({
+        active: pressed,
+      });
+    }
+  };
+
   render() {
+    const anotherCardActive = this.state.active === 'ANOTHER_CARD';
```
Comment (tbergq) [inline app/MMB/src/scenes/tripServices/insurance/paymentScene/PaymentScene.js]: isAnotherCardActive

**Comment by RobinCsl** [comment]
@tbergq 🚀

**Pull request merged**